nodes/nodes-sigstore-using.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="nodes-sigstore-using"]
+= Node metrics dashboard
```

Comment:
```suggestion
= Manage secure signature with sigstore
```

Reply:
Oh the joys of copy/paste. Sorry about that but thanks for catching it.